master.jinja2

```diff
@@ -16,6 +16,8 @@
 {% set AUTO_LOGIN_PASSWORD = AUTO_LOGIN_PASSWORD|default("") %}
 
 {% block global_settings %}
+reboot_to_fastboot: false
```

Comment:
maybe is should only apply to lxc templates?

Reply:
@mwasilew  - shall it be this file - projects/lkft/fastboot.jinja2 or fastboot.jinja2 ?

Reply:
either one. Probably better in fastboot.jinja2

Reply:
@mwasilew updated and pushed new patch.
Thanks for the review.